scripts/topiary

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Maybe too bold?

I did it impulsively but now feeling like it defeats the point of writing all this 2+3 compatible code.